Modules/Net/popup.swift

```diff
@@ -185,8 +185,10 @@ internal class Popup: NSStackView, Popup_p {
         
         let row: NSView = NSView(frame: NSRect(x: 0, y: 0, width: view.frame.width, height: Constants.Popup.separatorHeight))
         
+        let updatebutton_width = (LocalizedString("Refresh") as NSString).size(withAttributes: [NSAttributedString.Key.font: NSFont.systemFont(ofSize: 11, weight: .light)]).width
+        
         let button = NSButtonWithPadding()
-        button.frame = CGRect(x: view.frame.width - 44, y: 5, width: 44, height: 20)
+        button.frame = CGRect(x: view.frame.width - updatebutton_width, y: 5, width: updatebutton_width, height: 20)
```

Comment:
There is a little bit better way to do it.

First move `LocalizedString("Refresh")` to separate variable. Then you can use it in 3 places: for width, in the button title, and toolTip. I suppose the font for button text could be moved to a separate variable too.
Also, there is no reason for conversion to the `NSString`. `LocalizedString` returns `String` which confirms `NSString`.

PS: the code base of Stats use camel case (at least for variables)

Reply:
Thanks, I'll change this ASAP. Also think that an icon is a better approach